src/components/AddNewCustomer.tsx

```diff
@@ -0,0 +1,45 @@
+import { Button } from '@patternfly/react-core';
+import React, { useState } from 'react';
+
+function YourComponent() {
+  const [isModalOpen, setIsModalOpen] = useState(false);
+  const [newCustomer, setNewCustomer] = useState({
+
+    name: '',
+    age: '',
+    IsCool: '',
+  });
+  const addNewCustomer = async (customerData) => {
+    try {
+      const response = await CustomerApi.addCustomer(customerData);
```

Comment:
I'm curious what is leading you to try to add a new component? All the components that you need should be there.  If you noticing using the app you can see the list, click the button to add a new customer, fill in the details and hit submit.  

I see you're trying to call the Api here, but the right way to call it be to import it:
```
import { addNewCustomer, choosableColors, Customer, getCustomers } from 'src/api/CustomerApi';

```
and then call it:
```
addNewCustomer(newUser);
```

as imentioned before, you'd want to do this in the `onSubmit` function that you were working in with the previous change.